### DISCUSSION TITLE
Getting error: ./configs/operations-recommended is invalid

### DISCUSSION THREAD

**Discussion by nareshbhatia** [graphql-hive/graphql-eslint#1736] [Q&A] [answered]
When running ESLint, I am getting the following error:

```
ESLint: 8.44.0

Error: ESLint configuration in .eslintrc.js#overrides[1] » plugin:@graphql-eslint/operations-all » ./configs/operations-recommended is invalid:
 	- Unexpected top-level property "default".

 Referenced from: /my-project/node_modules/@graphql-eslint/eslint-plugin/cjs/index.js
```

Any idea why this is happening?

@graphql-eslint/eslint-plugin version: 3.20.0

**Comment by dimaMachina** [comment]
Do you have error in 3.19.0 version?

**Reply by nareshbhatia** [reply]
Version 3.19.0 works just fine. So something in version 3.20.0 is causing the break.

**Reply by dimaMachina** [reply]
hm, I have ci tests for examples that use cjs configs, and they don't fails on ci with your error
https://github.com/B2o5T/graphql-eslint/blob/096373df9471007b3e8d931d25bc56f3b1efac18/examples/multiple-projects-graphql-config/.eslintrc.cjs#L28

could you create a simple reproduction?

**Reply by nareshbhatia** [reply]
Hi @B2o5T, here's a simple reproduction:

https://github.com/nareshbhatia/graphql-eslint-example

Running `npm run lint` shows the error. Switching to version 3.19.0 in package.json eliminates the error.

**Reply by nareshbhatia** [reply]
@B2o5T, any update on this issue?

**Reply by dimaMachina** [reply]
Will fix it after the vacations, use previous working version

**Reply by nareshbhatia** [reply]
Will do. Have a great vacation.

**Reply by nareshbhatia** [reply, marked as answer]
This is fixed by version 3.20.1. See https://github.com/dimaMachina/graphql-eslint/pull/1750
